splunk.json

```diff
@@ -658,37 +664,51 @@
                 {
                     "data_path": "action_result.data.*.content.host",
                     "data_type": "string",
-                    "example_value": "test"
+                    "example_values": [
+                        "test"
+                    ]
                 },
                 {
                     "data_path": "action_result.data.*.content.info",
                     "data_type": "string",
-                    "example_value": "granted"
+                    "example_values": [
+                        "granted"
+                    ]
                 },
                 {
                     "data_path": "action_result.data.*.content.search",
                     "data_type": "string",
-                    "example_value": "search [| makeresults | eval myfield = \"test\" | table search] `notable` | table event_id"
```

Comment:
I feel like this is still a valuable example

Reply:
This example value has "|" in it and due to [this change](https://github.com/phantomcyber/dev-cicd-tools/blob/main/github-actions/start-release/build_docs.py#L94) "|" is not escaped which leads to rendering issue on the splunk soar instance hence, we have changed this example value.